articles/avoid-fix-flaky-tests.md

```diff
@@ -0,0 +1,73 @@
+---
+authorName: Adrian Herzog
+authorGithubUsername: adiherzog
+issue: 42
+title: How to avoid / deal with flaky system tests
```

Comment:
How to deal with flaky system tests?
    Or 
7 tips how to deal with flaky system tests?

Reply:
Thanks. Picking "How to deal with flaky system tests" for now.